src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -133,7 +133,7 @@ private List<MappingConfig> parseMappingConfig(Configuration conf) throws Invali
     /**
      * A method for troubleshooting purposes.
      */
-    private void logLoadedMappings() {
+    private void logDebugLoadedMappings() {
```

Comment:
We might still want the `if` protecting the contents of this file.

Reply:
done